hrabsence/tests/phpunit/CRM/HRAbsence/BAO/HRAbsenceTypeTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+use CRM_HRCore_Test_Fabricator_Contact as ContactFabricator;
+use CRM_HRAbsence_Test_Fabricator_HRAbsenceType as AbsenceTypeFabricator;
+use CRM_Hrjobcontract_Test_Fabricator_HRJobContract as HRJobContractFabricator;
+use CRM_Hrjobcontract_Test_Fabricator_HRJobLeave as LeaveFabricator;

```

Comment:
Why the HRJobContract fabricator was aliased to `HRJobContract` but the HRJobLeave was aliased to `LeaveFabricator`?